### DISCUSSION TITLE
Maintainer unresponsiveness

### DISCUSSION THREAD

**Discussion by twpayne** [twpayne/chezmoi#3292] [Maintainance]
chezmoi is an open source project with volunteer maintainers and zero expectations about any sort of responsiveness.

Only post here if you want to tell people that you'll be unresponsive but expect to be back.

**Comment by twpayne** [comment]
I'll be away for two weeks, roughly 14-28 October, with limited internet access and no way to make code changes. I will be unresponsive but expect to be back.

You can follow my adventures at https://www.instagram.com/tompayne3030/ if you want.

**Reply by Lockszmith-GH** [reply]
Enjoy

**Reply by arrrgi** [reply]
Safe travels!!
